Sources/BraintreeSEPADirectDebit/SEPADirectDebitAPI.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+class SEPADirectDebitAPI {
+    
+    private let apiClient: BTAPIClient
+    
+    @objc(initWithAPIClient:)
+    init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+
+    func createMandate(
+        sepaDirectDebitRequest: BTSEPADirectDebitRequest,
+        completion: @escaping (CreateMandateResult?, Error?) -> Void
+    ) {
+        let json: [String: Any] = [
+            "sepa_debit": [
+                "merchant_or_partner_customer_id": sepaDirectDebitRequest.customerID ?? "",
+                "mandate_type": sepaDirectDebitRequest.mandateType?.description ?? "",
+                "account_holder_name": sepaDirectDebitRequest.accountHolderName ?? "",
+                "iban": sepaDirectDebitRequest.iban ?? "",
+                "billing_address": [
+                    "address_line_1": sepaDirectDebitRequest.billingAddress?.streetAddress,
+                    "address_line_2": sepaDirectDebitRequest.billingAddress?.extendedAddress,
+                    "admin_area_1": sepaDirectDebitRequest.billingAddress?.locality,
+                    "admin_area_2": sepaDirectDebitRequest.billingAddress?.region,
+                    "postal_code": sepaDirectDebitRequest.billingAddress?.postalCode,
+                    "country_code": sepaDirectDebitRequest.billingAddress?.countryCodeAlpha2
```

Comment:
```suggestion
                    "address_line_1": billingAddress?.streetAddress,
                    "address_line_2": billingAddress?.extendedAddress,
                    "admin_area_1": billingAddress?.locality,
                    "admin_area_2": billingAddress?.region,
                    "postal_code": billingAddress?.postalCode,
                    "country_code": billingAddress?.countryCodeAlpha2
```
🧶 : cuts down on some of the repetition.

Reply:
Great suggestion! Will update this.